plugin/driver/pg_helper.go

```diff
@@ -370,6 +370,7 @@ func AddMetaconfig(domainID string, netID string, deviceID string, endpointID st
 			 "prop": {
 				"` + endpointID + `": {
 						"phy_address": "` + macaddr + `",
+						"ip_address": "` + ip + `",
```

Comment:
same change needed in above condition as well, in case of multiple interfaces IP won't be updated.